OpenTerm/Controller/ViewController.swift

```diff
@@ -339,6 +357,261 @@ extension ViewController: UIDocumentPickerDelegate {
 
 }
 
+extension ViewController: BookmarkViewControllerDelegate {
+    
+    /// The file name of the start directory bookmark. It is stored as a static
+    /// constant so we can access it when saving or loading the bookmark and
+    /// such that the settings view controller can access is as well.
+    static let bookmarkDirectory = ".bookmarks"
+    
+    
+    /// Gets the URLs of the bookmarks that were previously saved.
+    ///
+    /// - Returns: The URLs of the saved bookmarks. If something fails, the returned array will be empty.
+    func savedBookmarkURLs() -> [URL] {
+        /**
+         *  Get the document directory of the app.
+         */
+        if let dir = FileManager.default.urls(for: .documentDirectory,
```

Comment:
We already have a `DocumentManager` class that exposes an `activeDocumentsFolderURL`, which is either the value you're using here or the iCloud documents.

Is there a reason you are not using that? Do you want bookmarks only stored locally and not to sync with iCloud?